examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1018,7 +1058,16 @@ def main(args):
     )
 
     # Load scheduler and models
-    noise_scheduler = DDPMScheduler.from_pretrained(args.pretrained_model_name_or_path, subfolder="scheduler")
+    scheduler_type = determine_scheduler_type(args.pretrained_model_name_or_path, args.revision)
+    if scheduler_type == "EDMDPMSolverMultistepScheduler" or args.do_edm_style_training:
+        args.do_edm_style_training = True
+        noise_scheduler = EDMDPMSolverMultistepScheduler.from_pretrained(
```

Comment:
We should use `EDMEuler` here. For training, we should only recommend DDPM and Euler/EDMEuler. Rest are specifically designed for inference sampling.